### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.4 - es. 2 (14 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#290] [Esercizi M-Z]
![es.2](https://github.com/user-attachments/assets/581b2480-4941-4b27-b5be-84056b73c4f5)

**Comment by marcoseverini** [comment]
<img width="1037" alt="immagine" src="https://github.com/user-attachments/assets/211a403b-848f-4cd6-a084-6334286c2c0e">

**Comment by alem1105** [comment]
<img width="778" alt="375964328-51306f95-9894-40e5-9eb7-008e643a8d63" src="https://github.com/user-attachments/assets/63d5a6a5-da35-495d-a530-d2cff45eb7c3">
